testhelper/certificatetest.go

```diff
@@ -111,6 +131,21 @@ func getRSACertTuple(cn string, issuer *RSACertTuple) RSACertTuple {
 	return GetRSACertTupleWithPK(pk, cn, issuer)
 }
 
+func getRevokableRSACertTuple(cn string, issuer *RSACertTuple) RSACertTuple {
+	template := getCertTemplate(issuer == nil, true, cn)
+	template.OCSPServer = []string{"http://example.com/ocsp"}
+	return getRSACertTupleWithTemplate(template, issuer.PrivateKey, issuer)
+}
+
+func getRevokableRSAChainCertTuple(cn string, previous *RSACertTuple, index int) RSACertTuple {
+	template := getCertTemplate(previous == nil, true, cn)
+	template.BasicConstraintsValid = true
+	template.IsCA = true
+	template.KeyUsage = 0
```

Comment:
nit: Can we actually use contains instead of 0(see getCertTemplate method)?

Reply:
Makes sense. I'll update it from 0 to `x509.KeyUsageCertSign`